Classes/Emogrifier.php

```diff
@@ -323,6 +334,94 @@ protected function process(\DOMDocument $xmlDocument)
     }
 
     /**
+     * Applies $styles to $node.
+     *
+     * This method maps CSS styles to HTML attributes and adds those to the
+     * node.
+     *
+     * @param string[] $styles    the new CSS styles taken from the global styles
+     *                            to be applied to this node.
+     * @param \DOMNode $node      node to apply styles to.
+     *
+     * @return void
+     */
+    private function mapCssToHtmlAttributes(array $styles, \DOMNode $node)
+    {
+        foreach ($styles as $styleName => $styleValue) {
+            // Strip !important indicator
+            $styleValue = trim(str_replace('!important', '', $styleValue));
+            $this->mapCssToHtmlAttribute($styleName, $styleValue, $node);
+        }
+    }
+
+    /**
+     * Applies $styles to $node.
+     *
+     * This method maps a CSS rule to HTML attributes and adds those to the
+     * node.
+     *
+     * @param string $styleName   the name of the style rule to map.
+     * @param string $styleValue  the value of the style rule to map.
```

Comment:
According to the CSS nomenclature, those are called "property" and "value".

https://developer.mozilla.org/en-US/docs/Web/CSS/Syntax